packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.js

```diff
@@ -1,10 +1,68 @@
 import {LightningElement, api} from 'lwc';
+import LOCALE from '@salesforce/i18n/locale';
 
 export default class QuanticFacetValue extends LightningElement {
-  /** @type {import("coveo").FacetValue} */
   @api item;
   /** @type {boolean} */
   @api isChecked;
+  /** @type {string} */
+  @api variant = 'standard';
+  /** @type {(any) => string} */
+  @api formattingFunction;
+
+  Variants = {
+    StandardFacet: 'standard',
+    NumericFacet: 'numeric',
+    DateFacet: 'date',
+    Custom: 'custom'
+  };
+
+  get startDate() {
+    if (!this.isDateFacet) {
+      return undefined;
+    }
+    return new Intl.DateTimeFormat(LOCALE).format(
+      new Date(this.item.start)
+    );
+  }
+
+  get endDate() {
+    if (!this.isDateFacet) {
+      return undefined;
+    }
+    return new Intl.DateTimeFormat(LOCALE).format(
+      new Date(this.item.end)
+    );
+  }
+
+  get isStandardFacet() {
+    return this.variant === this.Variants.StandardFacet;
+  }
+
+  get isNumericFacet() {
+    const isNumericItem = this.item.start !== undefined && this.item.end !== undefined;
```

Comment:
```suggestion
    const isNumericItem = this.item?.start && this.item?.end;
```

Reply:
I had this before. If item.start is `0`, this returns false